packages/main/src/plugin/proxy.spec.ts

```
@@ -77,67 +71,69 @@ async function buildProxy(): Promise<ProxyServer> {
 test('check change from manual to system without proxy send event', async () => {
-  const configurationRegistry = getConfigurationRegistry(
-    ProxyState.PROXY_MANUAL,
-    `127.0.0.1:8080`,
-    undefined,
-    undefined,
-  );
-  const proxy = new Proxy(configurationRegistry, certificates);
-  await proxy.init();
+  await proxy?.setState(ProxyState.PROXY_MANUAL);
+  await proxy?.setProxy({
+    httpProxy: `127.0.0.1:8080`,
```

Biome (2.1.2)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Safe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)